packages/editor/src/components/widgets/input/IconInput.tsx

```diff
@@ -3,17 +3,30 @@ import type { IvyIcons } from '@axonivy/editor-icons';
 import IvyIcon from '../IvyIcon';
 import type { InputProps } from './Input';
 import Input from './Input';
-import { forwardRef } from 'react';
+import { forwardRef, useEffect, useRef } from 'react';
 
 export type IconInputProps = InputProps & {
   icon: IvyIcons;
+  initFocus?: boolean;
 };
 
-const IconInput = forwardRef<HTMLInputElement, IconInputProps>(({ icon, ...props }, forwardedRef) => {
+const IconInput = forwardRef<HTMLInputElement, IconInputProps>(({ icon, initFocus, ...props }, forwardedRef) => {
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    if (initFocus) {
+      const timer = setTimeout(() => {
+        inputRef.current?.focus();
+      }, 0);
+      return () => clearTimeout(timer);
+    }
+    return () => {};
```

Comment:
I think you don't need a return value if nothing is returned. -> remove return if initFocus is not true